pkgs/tools/misc/grub/2.0x.nix

```diff
@@ -110,6 +116,14 @@ stdenv.mkDerivation rec {
     sed -i $out/lib/grub/*/modinfo.sh -e "/grub_target_cppflags=/ s|'.*'|' '|"
   '';
 
+  passthru.tests = {
```

Comment:
Can we move the tests into another PR? I would like to have cross compiling support and they block that.

Reply:
Ok I can remove this. Can you explain why these would block cross-compiling? Obviously my whole motivation here is cross-compilation so I don't want to harm that :)

Reply:
I disagree. Testing goes before cross compiling. Especially for grub, and especially when a fix is provided in https://github.com/NixOS/nixpkgs/pull/112961

Reply:
> Can you explain why these would block cross-compiling?

Because they are failing.

> Especially for grub

also true.